app/utils/workflow.js

```diff
@@ -156,6 +160,20 @@ export default class WorkflowUtil {
           continue;
         }
 
+        // Check attribute filters
+        if (attributeFilter) {
+          const asset = WorkflowUtil.findAssetByUri(filteredAsset, entry.asset);
```

Comment:
This wasn't quite working.  I had an entrypoint file, I unchecked it in the list, and the file stayed visible.  I think the problem is that entry.asset has a relative URI, where filteredAsset (and its descendants) have an absolute URI.

You can use `filteredAsset.uri` as the base URI though to get you from relative -> absolute.  That is the approach taken when we convert the entries to the relative URI:
```
    const allDeps = WorkflowUtil.getAllDependencies(filteredAsset, filteredAsset.uri);
```